components/kyma-environment-broker/internal/lms/client.go

```diff
@@ -292,11 +289,6 @@ func (c *client) getCertificate(tenantID string, certID string) (cert string, fo
 		Cert string `json:"cert"`
 	}
 
-	err = json.Unmarshal(body, &certResponse)
```

Comment:
this change was done because first we need to check the response status, then unmarshall.